doc/reference/reference_lua/config.rst

```diff
@@ -351,178 +380,355 @@ config API
+    :dedent:
+
+Examples on GitHub: `tarantool_config_storage <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/centralized_config/instances.enabled/tarantool_config_storage>`_
+
+**Client API example:**
+
+..  literalinclude:: /code_snippets/snippets/centralized_config/instances.enabled/config_storage/myapp.lua
+    :language: lua
+    :start-at: storage_client:txn
+    :end-at: })
+    :dedent:
+
+Examples on GitHub: `config_storage <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/centralized_config/instances.enabled/config_storage>`_
+
+.. _config_storage_client_api_reference_is_connected:
+
+.. function:: <config.storage client>:is_connected()
+
+Show whether there is an active connection to a config.storage cluster.
+
+:return: true if connected, false on failure.
+:rtype:  boolean
+
+.. _config_storage_client_api_reference_close:
+
+.. function:: <config.storage client>:close()
+
+Close existing connection to a config.storage cluster. Connection objects are destroyed by the Lua garbage collector, just like any other objects in Lua, so an explicit destruction is not mandatory. However, since ``close()`` is a system call, it is good programming practice to close a connection explicitly when it is no longer needed, to avoid lengthy stalls of the garbage collector.
+
+Note connection to the config.storage will be re-established automatically if one of the remote config.storage client methods is called.
+
+.. _config_storage_client_api_reference_is_closed:
+
+.. function:: <config.storage client>:is_closed()
+
+Show whether the connection has been closed by a user with ``close()``.
+
+:return: true if closed, false on failure.
+:rtype:  boolean
+
+.. _config_storage_client_api_reference_reconnect:
+
+.. function:: <config.storage client>:reconnect()
+
+Reconnect to one of the config.storage instances after calling ``close()``.
+
+.. _config_storage_client_api_reference_on_disconnect:
+
+.. function:: <config.storage client>:on_disconnect(func)
+
+Define a trigger for execution when the established connection to a config.storage cluster instance is closed. This also includes the cases when the config.storage client loses connection to a specific instance but automatically re-establishes it to another available one. Execution stops after a config.storage client is explicitly closed, or once the Lua garbage collector removes it.
+
+.. _config_storage_client_api_reference_watch:
+
+.. function:: <config.storage client>:watch(path_or_key, func)
+
+Subscribe to config.storage events broadcast. You might use one of the following events.
+
+* ``info`` is an event corresponding to the update on the client information.
+* ``/path/to/key`` or ``/prefix/to/watch/`` is an event that corresponds to the updates of the specific key or of multiple keys having the specified prefix.
+
+:param string key: a key name of an event to subscribe to
+:param function func:  a callback to invoke when the key value is updated
```

Comment:
What the callback receives as arguments (in case of `info`, in case of a key, in case of a prefix)?